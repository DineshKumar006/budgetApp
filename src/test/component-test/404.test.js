//  expect(tree).toMatchSnapshot();

import React from 'react'

import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json';
import PageNotFound from '../../components/404Page/404Page'



test('404 page test',()=>{
    const wrapper=shallow(<PageNotFound/>);

    expect(toJSON(wrapper)).toMatchSnapshot();
})

test('spy function',()=>{
    const onSubmitSpy=jest.fn();
    onSubmitSpy('dinesh','latha')
    expect(onSubmitSpy).toHaveBeenCalledWith('dinesh','latha')
})