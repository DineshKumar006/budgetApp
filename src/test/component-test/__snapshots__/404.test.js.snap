// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page test 1`] = `
<div>
  <h1>
    404! page not found
  </h1>
  <NavLink
    to="/"
  >
    Go to dashboard
  </NavLink>
</div>
`;
